hieradata/common.yaml

```diff
@@ -1542,6 +1542,8 @@ monitoring::checks::smokey::features:
     feature: router
   check_search:
     feature: search
+  check_service_manual:
+    feature: service_manual
```

Comment:
This is the Carrenza file, whereas the commit message says AWS?

Reply:
My bad - commit message fixed